src/05_lists.py

```diff
@@ -8,22 +8,28 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE
+x.pop(4)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
 # YOUR CODE HERE
+x.insert(4, 99)
 print(x)
 
 # Print the length of list x
 # YOUR CODE HERE
+print("This list's length is: " + str(len(x)))
 
 # Print all the values in x multiplied by 1000
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+print("The values in x multiplied by 1000 is: " + str(sum(x) * 1000))
```

Comment:
This is a totally valid interpretation of those words! It should absolutely say "each value" or something. Just so you know, what they wanted was something like:
`for e in x:`
`  print(e * 1000)`